src/transforms/boxShadow.js

```diff
@@ -28,10 +28,8 @@ module.exports = (tokenStream) => {
         tokenStream.expect(SPACE);
         blurRadius = tokenStream.expect(LENGTH);
       }
-    } else if (color === undefined && (
-      tokenStream.matchesFunction() || tokenStream.matches(WORD)
-    )) {
-      color = String(tokenStream.lastValue);
+    } else if (tokenStream.matches(COLOR)) {
```

Comment:
I think we need to check `color === undefined` to stop `box-shadow: 0 0 0 red yellow green blue`

Reply:
ok, should I add a test case for that?

Reply:
I added a test case and a fix for that. :)